apps/examples/testcase/le_tc/kernel/tc_libc_timer.c

```diff
@@ -339,7 +343,7 @@ static void tc_libc_timer_strftime(void)
 	TC_ASSERT_EQ("strftime", strcmp(buffer, "June"), 0);
 
 	/* Check the century number (year/100) as a 2-digit integer */
-	strftime(buffer, BUFF_SIZE, "%C", &st_time);
+	strftime(buffer, BUFF_SIZE, "%y", &st_time);
```

Comment:
Please set the testcase sequence by alphabet order.

Reply:
modify